xml/article_setup.xml

```diff
@@ -25,7 +25,7 @@
 
    <para>
     To meets this requirement, &slerte; offers several options for CPU and I/O
-    scheduling, CPU shielding and for setting CPU affinities to processes.
+    scheduling, CPU shielding, and for setting CPU affinities to processes.
```

Comment:
Iirc, our spell-checker (rspell) really does not like the plural of `affinity` somehow. Are there good reasons for that?

Reply:
None that I know of; it is a legitimate plural. Citation:
https://www.merriam-webster.com/dictionary/affinity

Reply:
Thanks!